src/stats/timeline.R

```diff
@@ -39,7 +39,22 @@ SELECT m.runId, m.trialId, m.criterion, m.value
 result <- dbFetch(qry)
 dbClearResult(qry)
 dbCommit(rebenchdb)
+convert_array <- function(x) {
+    x <- gsub("(^\\{|\\}$)", "", x)
+    strsplit(x, split = ",")
+}
+
+convert_double_array <- function(x) {
+    sapply(convert_array(x), as.double)
+}
+
+result <-
+   result %>% 
+    collect() %>%
+    mutate(value = convert_double_array(value)) 
+
 
+result <- unnest(result, cols = c(value))   
```

Comment:
Please do all necessary transformations in the `rebenchdb.R` file, where I assume is duplicated code from this, or at least a close variant.

It would also be good to know which version is faster, the one doing it in R or the one doing it in the database.